src/js/edu-benefits/pages/applicantInformation.js

```diff
@@ -0,0 +1,57 @@
+import _ from 'lodash/fp';
+
+import * as currentOrPastDate from '../../common/schemaform/definitions/currentOrPastDate';
+import * as fullName from '../../common/schemaform/definitions/fullName';
+import * as ssn from '../../common/schemaform/definitions/ssn';
+
+import definitions from 'vets-json-schema/dist/definitions.json';
+
+import { relationshipLabels, genderLabels } from '../utils/helpers';
+
+const defaultProps = [
+  'relativeFullName',
+  'relativeSocialSecurityNumber',
+  'relativeDateOfBirth',
+  'gender',
+  'relationship'
+];
+
+export default function applicantInformation(propNames = defaultProps) {
+  const possibleProperties = {
```

Comment:
The way I did this for the school selection page was to pass in a schema and pick the properties from that. I think that avoids potentially getting out of sync with one of the form schemas, because if you include a field that isn't there, it won't show up, instead of getting included and causing a backend validation errors. And if you include `ui:order`, it'll throw a helpful error in the console, even.